CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.3.0] - 2021-02-01
```

Comment:
Shouldn't this be 4.2.1? Also changelog missing.

Reply:
https://github.com/giantswarm/devctl/pull/217